src/ocm/components/clusterWizard/ClusterPlatformIntegrationHint.tsx

```diff
@@ -27,13 +29,17 @@ export const integrationPlatformLinks: Record<SupportedPlatformType, string> = {
 export const ClusterPlatformIntegrationHint = ({
   clusterId,
   platformType,
+  openshiftVersion,
 }: ClusterPlatformIntegrationHintProps) => {
   const { isPlatformIntegrationSupported, supportedPlatformIntegration } =
     useClusterSupportedPlatforms(clusterId);
-
+  const featureSupportLevels = useFeatureSupportLevel();
+  const isNutanixFeatureSupported =
+    featureSupportLevels.isFeatureSupported(openshiftVersion || '', 'NUTANIX_INTEGRATION') ?? false;
   const canIntegrateWithPlatform =
```

Comment:
This condition is a bit hard to read, but I'm not sure it's 100% correct now.
The OR now will return true if the second part returns true, even if the previous condition is false.

In particular, is the `cluster.platformType` is already `nutanix` or `vsphere`, we don't want the banner to be displayed, because that means that the integration has already been selected.